lib/resources/r4/encounter.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/R4/encounter-definitions.html#Encounter
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: Identifier(s) by which this encounter is known.
+  children:
+
+  - name: type
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    binding:
+      description: The type used to determine which identifier to use for a specific purpose.
+      terminology:
+      - display: Identifier Type Codes
+        system: http://hl7.org/fhir/identifier-type
+        info_link: http://hl7.org/fhir/R4/valueset-identifier-type.html
+      - display: v2 Identifier Type
+        system: http://terminology.hl7.org/CodeSystem/v2-0203
+        info_link: http://hl7.org/fhir/R4/valueset-identifier-type.html
+      - display: Millennium Encounter AliasType
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/319
+        info_link: https://fhir.cerner.com/millennium/R4/proprietary-codes/#code-set-319-encounter-alias-type
```

Comment:
seems to work either way, but would be better to use r4 instead of R4

Reply:
R4 -> r4